includes/payment-methods/class-upe-split-payment-gateway.php

```diff
@@ -48,6 +49,13 @@ class UPE_Split_Payment_Gateway extends UPE_Payment_Gateway {
 	 */
 	protected $stripe_id;
 
+	/**
+	 * WC_Payments_Localization_Service instance.
+	 *
+	 * @var WC_Payments_Localization_Service
+	 */
+	protected $localization_service;
```

Comment:
Isn't it inherited?

Reply:
Same as previous, changed it here https://github.com/Automattic/woocommerce-payments/pull/6920/commits/3dc1eb4d18404c33265e1217b9157f87c679920e